qiita_pet/support_files/doc/source/dev/plugins.rst

```diff
@@ -0,0 +1,81 @@
+.. _plugins:
+
+.. index :: plugins
+
+Plugins
+=======
+
+Qiita plugins allow to extend the functionality of Qiita by adding new data
+formats and/or new processing pipelines. The list of officially supported
+plugins can be found in the `Qiita Spots <https://github.com/qiita-spots>`__
+GitHub organization.
+
+Overview of Qiita's plugin system
+---------------------------------
+
+Qiita's plugin system has been designed to increase Qiita's flexibility to
+process -omics datasets, while allowing plugin developers to create plugins
+using the programming language that they are more comfortable with.
+
+In order to achieve this flexibility, Qiita represents the different -omics
+datasets using ``Artifacts``. An ``Artifact`` is a file or set of files that
+conceptually represents a dataset. However, Qiita does not understand the
```

Comment:
However, Qiita ... -> Note that Qiita is not Artifact or Pipeline aware, thus its is up to the Qiita Plugins ...